nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,351 @@
+        }
+
+        .flex-layout nuxeo-card {
+          flex: 1 0 calc(30% - 2em);
+          margin: 0 8px 16px;
+          text-align: center;
+          box-sizing: content-box;
+        }
+
+        nuxeo-card .alignLeft {
+          text-align: left;
+        }
+
+        nuxeo-data-table {
+          height: 550px;
+        }
+
+        chart-line,
+        chart-pie {
+          margin: 25px auto 0 auto;
+          width: 100% !important;
+          display: block;
+          font-size: 0.8rem;
+        }
+
+        @media (max-width: 1024px) {
+          .flex-layout nuxeo-card {
+            flex: 1 0 calc(100% - 2em);
+          }
+        }
+
+        .mainRetentionInfo {
+          color: #7e90a5;
+          text-transform: uppercase;
+          margin-inline-end: 8px;
+        }
+
+        .retentionCount {
+          font-weight: bold;
+        }
+
+        .chartBar {
+          text-align: center;
+        }
+      </style>
+      <nuxeo-page>
+        <div slot="header">
+          <span>[[i18n('retention.analytics')]]</span>
+        </div>
+        <!-- Number of documents under retention -->
+        <nuxeo-repository-data
+          start-date="[[_formatDate(startDate)]]"
+          end-date="[[_extendEndDate(endDate)]]"
+          date-field="dc:created"
+          where='{"ecm:isRecord": "true"}'
+          metrics="cardinality(ecm:uuid)"
+          data="{{activeRetentionCount}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <!-- Number of Files under legal hold -->
+        <nuxeo-repository-data
+          start-date="[[_formatDate(startDate)]]"
+          end-date="[[_extendEndDate(endDate)]]"
+          date-field="dc:created"
+          ecm-mixin-type="HasRelatedText"
+          where='{"ecm:hasLegalHold": "true"}'
+          metrics="cardinality(ecm:uuid)"
+          data="{{legalHoldCount}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <nuxeo-card class="infoCard">
+          <div class="rangeSelector">
+            <span class="mainRetentionInfo"
+              >[[i18n('retentionAnalytics.underActiveRetention')]]
+              <span class="retentionCount">[[activeRetentionCount]]</span>
+            </span>
+            <span class="mainRetentionInfo"> | </span>
+            <span class="mainRetentionInfo"
+              >[[i18n('retentionAnalytics.legalHoldCount')]]
+              <span class="retentionCount">[[legalHoldCount]]</span>
+            </span>
+          </div>
+          <div class="rangeSelector">
+            <div class="dateLabel">
+              <label>[[i18n('retentionAnalytics.from')]]</label>
+            </div>
+            <nuxeo-date-picker value="{{startDate}}"></nuxeo-date-picker>
+            <div class="dateLabel">
+              <label>[[i18n('retentionAnalytics.to')]]</label>
+            </div>
+            <nuxeo-date-picker value="{{endDate}}"></nuxeo-date-picker>
+          </div>
```

Comment:
Quick suggestion on the markup organization and the introduction of the property `for` on the label for better accessibility:
```suggestion
          <div class="rangeSelector">
            <div>
              <label class="dateLabel" for="startDate">[[i18n('retentionAnalytics.from')]]</label>
              <nuxeo-date-picker id="startDate" value="{{startDate}}"></nuxeo-date-picker>
            </div>
            <div>
              <label class="dateLabel" for="endDate">[[i18n('retentionAnalytics.to')]]</label>
              <nuxeo-date-picker id="endDate" value="{{endDate}}"></nuxeo-date-picker>
            </div>
          </div>
```

Reply:
Great suggestion! Thanks!